params.yml

```
@@ -179,3 +182,20 @@ args:
 
   ## Tools versions
   polycli_version: v0.1.42
+
+  ## blutgang configuration.
+  blutgang_image: makemake1337/blutgang:latest
```

Use a fixed version instead of `latest` just in case a new release breaks everything.

I think `0.3.5` is the latest one: https://github.com/rainshowerLabs/blutgang/releases

```suggestion
  blutgang_image: makemake1337/blutgang:0.3.5
```